### ISSUE TITLE
pydoclint uses variable instead of exception itself

### ISSUE THREAD

**Issue by jsh9** [jsh9/pydoclint#175]
It was originally reported here: https://github.com/jsh9/pydoclint/issues/174#issuecomment-2387881545

Code example (Google style docstring):

```python
def myFunc():
    """
    Do something

    Raises:
        IOError: If something is wrong
    """
    try:
        3 + 3
    except IOError as e:
        raise e
```

Violation message:

```
'DOC503: Function `myFunc` exceptions in the "Raises" section in the '
 'docstring do not match those in the function body Raises values in the '
 "docstring: ['IOError']. Raised exceptions in the body: ['e']."
```

(There should have been no violations.)

**Comment by jsh9** [comment]
Hi @Amar1729 , could you take a look at this issue and see if there is a fix?

Thanks!

**Comment by Amar1729** [comment]
hm, thought I had a test for this case. yeah I can take a look.

**Comment by Gnosnay** [comment]
any progress on this?

**Comment by jsh9** [comment]
Hi @Amar1729 and @Gnosnay , any help is greatly appreciated!

**Comment by Amar1729** [comment]
@jsh9 i had some unexpected life stuff come up and have been super busy, my bad. just got a chance to look at this, turned out to be a relatively simple fix. Let me know how the PR looks (and of course, if you can think of additional edge cases feel free to add them; clearly there are many related to extracting raised exception names from source code ;))

**Comment by jsh9** [comment]
Hi @Amar1729 , thank you so much for making the change! 

I can't think of any more edge cases for now, so I merged your current test cases as is.

**Comment by jsh9** [comment]
The fix is published in 0.5.11.
